test/contracts/ens/ens_subdomains.js

```diff
@@ -81,20 +78,20 @@ contract('ENSSubdomainRegistrar', ([_, apmOwner, notOwner]) => {
         await registrar.createName(holalabel, apmOwner, { from: apmOwner })
         await registrar.deleteName(holalabel, { from: apmOwner })
 
-        assert.equal(await ens.owner(holanode), ZERO_ADDR, 'should have reset name')
+        assert.equal(await ens.owner(holanode), ZERO_ADDRESS, 'should have reset name')
     })
 
     it('can delete names registered to itself', async () => {
         await registrar.createName(holalabel, registrar.address, { from: apmOwner })
         await registrar.deleteName(holalabel, { from: apmOwner })
 
-        assert.equal(await ens.owner(holanode), ZERO_ADDR, 'should have reset name')
+        assert.equal(await ens.owner(holanode), ZERO_ADDRESS, 'should have reset name')
     })
 
     it('fails if initializing without rootnode ownership', async () => {
         const ens = await ENS.new()
-        const newRegProxy = await AppProxyUpgradeable.new(dao.address, hash('apm-enssub.aragonpm.eth'), EMPTY_BYTES)
-        const newReg = ENSSubdomainRegistrar.at(newRegProxy.address)
+        const newRegProxy = await AppProxyUpgradeable.new(dao.address, hash('apm-enssub.aragonpm.eth'), '0x')
```

Comment:
Interesting, we may want to play with remix or debug this a bit more.

Theoretically `0x00` should be a bytes array of length 0, but it looks like it may not be interpreted correctly in `AppProxyBase`.